src/Sdk/DTWebApi/WebApi/Telemetry.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Runtime.Serialization;
+
+namespace GitHub.DistributedTask.WebApi
+{
+    /// <summary>
+    /// Information about a job run on the runner
+    /// </summary>
+    [DataContract]
+    public class Telemetry
```

Comment:
What properties should a generic telemetry have?

Reply:
Lets start simple with just Message for now.